gammapy/maps/region/geom.py

```diff
@@ -472,7 +472,7 @@ def to_image(self):
         region : `~RegionGeom`
             RegionGeom without any non-spatial axes.
         """
-        return self._init_copy(axes=None)
+        return self._init_copy(region=self.region, wcs=self.wcs, axes=None)
```

Comment:
A naive question: does it work with healpix?

Reply:
Region geoms always work with a tangential WCS, there is no support for healpix...